server/test/Helpers.cc

```diff
@@ -689,17 +689,6 @@ string makeItemOutput(const ItemInfo &itemInfo)
 	    static_cast<int>(itemInfo.valueType),
 	    itemInfo.unit.c_str());
 
-	// Item category
```

Comment:
Ah, I got it. `items` tables no longer has `categoryNames`.
We should integrate `item_categories` result in this assertion, but this task is not current PR's task.